### ISSUE TITLE
BUG: Bun does not respect `logLevel = "error"` when ran through package.json script

### ISSUE THREAD

**Issue by nikivdev** [oven-sh/bun#5557]
### What version of Bun is running?

1.0.3+b651b16fdd959b349c09ec5d5056229dcd12c86b

### What platform is your computer?

Darwin 22.6.0 arm64 arm

### What steps can reproduce the bug?

In this repo: https://github.com/learn-anything/learn-anything.xyz

I have bunfig.toml that looks like this:

https://github.com/learn-anything/learn-anything.xyz/blob/main/bunfig.toml

```
logLevel = "error"
```

I then run a file inside with bun like so

```
bun db
```

I see:

<img width="386" alt="image" src="https://github.com/oven-sh/bun/assets/6391776/86fbac2b-7f05-41bb-93a3-f8f7e20ed610">

`[0.20ms] ".env"` should not be there!

If I run the file with just bun directly like so:

```
❯ bun --watch edgedb/cli/cli.ts
test
```

There is no `[0.20ms] ".env"`

Basically when ran through package.json, Bun ignores the setting I have set. 

Please fix it as I really dislike seeing `[0.18ms] ".env"` every time I run the script. And I use package.json scripts often.

**Comment by colinhacks** [comment]
This has been fixed in v1.0.3: https://bun.sh/blog/bun-v1.0.3#message-is-now-silent-by-default-0-5ms-env
